flutter_local_notifications/README.md

```diff
@@ -43,37 +43,208 @@ A cross platform plugin for displaying local notifications.
 * [iOS] Request notification permissions and customise the permissions being requested around displaying notifications
 * [iOS] Display notifications with attachments
 
-Note that this plugin aims to provide abstractions for all platforms as opposed to having methods that only work on specific platforms. However, each method allows passing in "platform-specifics" that contains data that is specific for customising notifications on each platform. This approach means that some scenarios may not be covered by the plugin. Developers can either fork or maintain their code for showing notifications in these situations. Note that the plugin still under development so expect the API surface to change over time.
 
-**IMPORTANT NOTES**:
 
-* Recurring notifications on Android use the [Alarm Manager](https://developer.android.com/reference/android/app/AlarmManager) API. This is standard practice but does mean the delivery of the notifications/alarms are inexact and this is documented Android behaviour as per the previous link. Note that it's been reported that Samsung's implementation of Android has imposed a maximum of 500 alarms that can be scheduled via this API and exceptions can occur when going over the limit
-* iOS has a limit on how many pending notifications it allows. This is a limit imposed by iOS where it will only keep 64 notifications that will fire the soonest
-* *Known issue*: There is a known issue with handling daylight savings for scheduled notifications. This functionality may be deprecated to be replaced by another that only deals with elapsed time since epoch instead of a date.
+## ⚠ Caveats and Limitations
+Note that this plugin aims to provide abstractions for all platforms as opposed to having methods that only work on specific platforms. Each method allows passing in "platform-specifics" that contains data that is specific for customising notifications on each platform. This approach means that some scenarios may not be covered by the plugin. Feel free to fork or maintain your code for showing notifications in these situations.
 
-## Screenshots
+##### firebase_messaging
+Please bump `firebase_messaging` to 6.0.13 or higher (and update the code accordingly) to resolve any issues.
```

Comment:
I'm not aware of the specifics of the conflict when using both plugins together. The original text was:
```
Previously, there were issue that prevented this plugin working properly with the firebase_messaging plugin. This meant that . Version 6.0.13 of firebase_messaging should resolve this issue so please bump your firebase_messaging dependency and follow the steps covered in firebase_messaging's readme file.
```

I wasn't sure whether it was just 1 issue or multiple issues, I just know that some people reported issues.

How about
```
Using this plugin with `firebase_messaging` versions before 6.0.13 has issues, where the callbacks from each plugin may not be invoked. Please use `firebase_messaging` versions 6.0.13 or higher (and update the code accordingly) to resolve any issues.
```

Reply:
If the primary goal of the PR is to rearrange the sections then I would suggest to keep the original wording. If further changes are made then it should keep the context that was originally there. Happy to answer questions on what the details are if needed

> Using this plugin with firebase_messaging versions before 6.0.13 has issues, where the callbacks from each plugin may not be invoked. Please use firebase_messaging versions 6.0.13 or higher (and update the code accordingly) to resolve any issues.

Mentioning the range of version of numbers that will have problems with this plugin and the range of version numbers that work a bit redundant. Could recommend to versions 6.0.13 or higher to avoid issues

Reply:
Good point, I have replaced this with the original text